assets/blueprints/faucet/src/lib.rs

```diff
@@ -9,14 +9,14 @@ mod faucet {
     }
 
     impl Faucet {
-        pub fn new(preallocated_address_bytes: [u8; 30], bucket: Bucket) -> Global<Faucet> {
+        pub fn new(address_ownership: Owned<AnyComponent>, bucket: Bucket) -> Global<Faucet> {
```

Comment:
`address_ownership` is a little bit of a weird name. Can we change to something like `owned_address`